api/sample-code.md

```diff
@@ -59,4 +58,7 @@ $jsonData = json_decode($response, true);
 
 // Dump array structure for inspection
 var_dump($jsonData);
+
+?>
```

Comment:
The closing PHP tag is not required in PHP files so could you remove these 3 new lines at the end of the code sample for us? Thanks!